app/controllers/dpa_exceptions_controller.rb

```diff
@@ -1,17 +1,20 @@
 class DpaExceptionsController < InheritedResources::Base
+  devise_group :logged_in, contains: [:user, :admin]
+  before_action :authenticate_logged_in!
 
-  def destroy
+  def index
+    @dpa_exceptions = DpaException.active
+  end
+
+  def archive
     @dpa_exception = DpaException.find(params[:id])
```

Comment:
I believe that the conditional is missing. On line 12 you want `if @dpa_exception.archive` so it can √ for success right?